### PR TITLE
Infrastructure refactoring Part 1

### PR DESCRIPTION
**Pull request by tjaeschke**
Fixes for issues #24, #40, #42, #12, #4

### CODE REVIEW

**Comment by tjaeschke** [comment]
Fixes for issues #24, #40, #42, #12, #4

**Comment by clemensutschig** [comment]
@tjaeschke how does this work with the github.io website

**Comment by tjaeschke** [comment]
Not yet. There will be a new guide for using this, because I have automated everything as far as possible and only the Atlassian configuration will be left. There are two main scripts handling various sub scripts fir automated setup. We have kicked Minishift and use an AiO Cluster based on cluster up instead. It is still work in progress, but I thought it would be good, if there is an insight what's happening.


OPITZ CONSULTING Deutschland GmbH
Standort Gummersbach
Torsten Jaeschke - Solution Architect
AWS Certified Solutions Architect - Professional
Kirchstraße 6 | D-51647 Gummersbach

Telefon: +49 2261 6001-1164 | Telefax: +49 2261 6001-4200
Mobil: +49 173 9002713
torsten.jaeschke@opitz-consulting.com
www.opitz-consulting.com<http://www.opitz-consulting.com>

Geschäftsführer: Dr. Sarah Opitz, Tom Gansor, Peter Menne, Torsten Schlautmann
HRB-Nr. 76555 Amtsgericht Köln

________________________________

File: jenkins/slave-base/Dockerfile.centos7 [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM openshift/jenkins-slave-base-centos7
+FROM openshift/jenkins-slave-base-centos7:v3.11
```
Comment (michaelsauter) [inline jenkins/slave-base/Dockerfile.centos7]: I don't think we should do this. It basically means we only support 3.11. See how we did this in #91.

Comment (tjaeschke) [reply jenkins/slave-base/Dockerfile.centos7]: I will take a look.

Comment (tjaeschke) [reply jenkins/slave-base/Dockerfile.centos7]: Reworked it, accordingly to #91

**Comment by tjaeschke** [comment]
The documentation on opendevstack.github.io is still missing, I didn't get it this week, but the old way to install will still work.

Due to the fact, that the documentation doesn't fit at the moement this shouldn't prevent it from merging.

Feedback is very welcome.

**Comment by michaelsauter** [comment]
Too many changes for feedback :)

Would have been nice to cut this into more manageable pieces ... anyway. Should this go into 1.1 or not?

**Comment by tjaeschke** [comment]
> Too many changes for feedback :)
> 
> Would have been nice to cut this into more manageable pieces ... anyway. Should this go into 1.1 or not?

Most changed files result from adding new ansible roles for the AiO cluster, that is now used instead of a Minishift. It would be good, if we have this in 1.1

File: .gitignore [hunk 2]
```diff
@@ -1,3 +1,15 @@
 .project
 .vscode
 *.env
+/Finished
+/Package
+/atlcon
+/vagrant
+/atlassian
```
Comment (michaelsauter) [inline .gitignore]: Those characters at the end look weird?

Comment (tjaeschke) [reply .gitignore]: I have removed them. This have been small computer icons. I don't know how they got there. Please approve again.

**Pull request merged**